UnityProject/Assets/Scripts/HealthV2/Living/PolymorphicSystems/Bodypart/HungerComponent.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using Chemistry;
+using HealthV2;
+using HealthV2.Living.PolymorphicSystems;
+using HealthV2.Living.PolymorphicSystems.Bodypart;
+using UnityEngine;
+using UnityEngine.Serialization;
+
+public class HungerComponent : BodyPartComponentBase
+{
+	/// <summary>
+	/// Modifier that multiplicatively reduces the efficiency of the body part based on damage
+	/// </summary>
+	[Tooltip("Modifier to reduce efficiency when the character gets hungry")] [NonSerialized]
+	public Modifier HungerModifier = new Modifier();
+	//TODO add to Body part
+
+
+	/// <summary>
+	/// The nutriment reagent that this part consumes in order to perform tasks
+	/// </summary>
+	[Tooltip("What does this live off?")] [SerializeField]
+	public Reagent Nutriment;
+
+
+	/// <summary>
+	/// The amount of of nutriment to consumed each tick as part of passive metabolism
+	/// </summary>
+	[NonSerialized] //Automatically generated runtime
+	public float PassiveConsumptionNutriment = 0.00012f;
```

Comment:
so are we still doing per body part hunger?

Reply:
well  out of scope of the re-factor

Reply:
fair enough